task/show-sbom/0.1/show-sbom.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-sbom
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
```

Comment:
Still use hacbs?

Reply:
It's present in all labels. I think it should be a separate PR that deletes HACBS stuff (and not only from label)